src/calculateRentalCost.js

```diff
@@ -4,7 +4,23 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const discount1 = 20;
+  const discount2 = 50;
+  const rentPerDays = 40;
+  let total = rentPerDays * days;
+
+  if (days < 3) {
+  }
+
+  if (days >= 3 && days < 7) {
```

Comment:
Consider using constants instead of magic numbers. https://github.com/mate-academy/js_task-transportation-on-vacation/blob/master/checklist.md 
You can use a checklist to follow